pkg/ipam/node.go

```diff
@@ -763,17 +763,7 @@ func (n *Node) syncToAPIServer() (err error) {
 // update is a helper function for syncToAPIServer(). This function updates the
 // CiliumNode resource spec or status depending on `status`. The resource is
 // updated from `origNode` to `node`.
```

Comment:
Can you update this function documentation so that we can easily know what will happen to `origNode` and `node` if err is or is not nil?

Reply:
Done.